test/cases/manifest_tests.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+# Get OS details.
+source /etc/os-release
+ARCH=$(uname -m)
+DISTRO_CODE="${DISTRO_CODE:-${ID}-${VERSION_ID//./}}"
+
+# get manifest-db at the version specified for this arch+distro
+MANIFEST_DB_COMMIT=$(cat Schutzfile | jq -r '.["'"${ID}-${VERSION_ID}"'"].dependencies."manifest-db".commit')
+MANIFEST_DB_REPO="https://github.com/osbuild/manifest-db"
+git clone "$MANIFEST_DB_REPO" manifest-db
+cd manifest-db
+git checkout "$MANIFEST_DB_COMMIT"
+
+# update selinux labels for the image-info tool
+OSBUILD_LABEL=$(matchpathcon -n $(which osbuild))
+chcon $OSBUILD_LABEL tools/image-info
+
+# run the tests from the manifest-db for this arch+distro
+echo "Running the osbuild-image-test for arch $ARCH and ditribution $DISTRO_CODE"
+sudo tools/osbuild-image-test --arch=$ARCH --distro=$DISTRO_CODE
```

Comment:
I still think that this should not live in the `test/cases` directory since it is an integration test that is run "on top of" osbuild, like the current composer tests. Just move it all into `schutzbot`.